programs/survival/fakeplayer_inv_editor.sc

```diff
@@ -0,0 +1,107 @@
+__config()->{
+    'scope'->'global',
+    'stay_loaded'->true
+};
+create_datapack('invupd', 
+    {
+        'readme.txt' -> ['this data pack is created by scarpet','please dont touch it'],
+        'data' -> {
+            'chyx/advancements/xd.json'-> {
```

Comment:
Is this path notation an intended one?
I saw it only in a single example in the docs, but I got some issues while using it in another script.

Reply:
IDK if it is intended.

that's not written by me.

maybe you should ask gnembon.

Reply:
Gembon said it is not an intended way.
It doesn't work sometimes

Reply:
@BisUmTo is it now acceptable?